src/control/lib/spdk/SConscript

```diff
@@ -14,15 +14,9 @@ def scons():
     senv.AppendUnique(CPPPATH=[join(Dir('.').srcnode().abspath, "include")])
     senv.AppendUnique(OBJPREFIX='s_')
 
-    # SPDK related libs
-    libs = ['spdk_env_dpdk', 'spdk_thread', 'spdk_bdev', 'spdk_copy']
-    libs += ['rte_mempool', 'rte_mempool_ring', 'rte_bus_pci']
-    libs += ['rte_pci', 'rte_ring', 'rte_mbuf', 'rte_eal', 'rte_kvargs']
-    libs += ['spdk_bdev_aio', 'spdk_bdev_nvme', 'spdk_bdev_malloc']
-    libs += ['spdk_conf', 'spdk_blob', 'spdk_nvme', 'spdk_util']
-    libs += ['spdk_json', 'spdk_jsonrpc', 'spdk_rpc', 'spdk_trace']
-    libs += ['spdk_sock', 'spdk_log', 'spdk_notify', 'spdk_blob_bdev']
-    libs += ['spdk_vmd']
+    # Explicitly link RTE & SPDK libs for CGO access
+    libs = ['spdk_env_dpdk', 'spdk_nvme', 'spdk_vmd', 'rte_mempool']
+    libs += ['rte_mempool_ring', 'rte_bus_pci', 'nvme_control']
```

Comment:
there are only defined twice in control/lib/spdk so I think it's okay for the moment

Reply:
Just need to make sure if the list changes again that we get it in all places. I saw another version of the same list in the src/control/SConscript, too (ldopts, it's a string but we could construct it from an array). I was thinking it could create the array and generate the ldopts string from that, and export the value for these sub-SConscripts. Needn't be addressed now but I think it will make things easier for our future selves.

Reply:
https://jira.hpdd.intel.com/browse/DAOS-7960 includes attempting to switch to using the lspdk linker script which will avoid needing a list and hopefully resolve this